.eslintignore

```diff
@@ -2,4 +2,4 @@ node_modules
 dist
 .rpt2_cache
 packages/design/colors.js
-plop/templates
+packages/generators/templates
```

Comment:
A couple of thoughts on naming. 
Because people use the `npm run plop` command to invoke it, they will be familiar with that term. I assume that we're using `generators` because we may want to change the implementation (i.e. plop) in the future? Should we change the npm command name as well? Don't get me wrong, I like plop 💩, but if they are importing a `generator`, it seems misaligned to still run `plop`. 

Is "generators" too generic? Would "componentGenerator" be more clear? (Both here in the code and when importing into something like Jobber.)